doc/porting/board_porting.rst

```diff
@@ -120,3 +120,25 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+
+Default board configuration
+***************************
+
+When porting a board, you must provide its default configuration. This default
```

Comment:
Should this be "when porting to a board"?

Reply:
Or

    When porting to a board, you must provide the board's default
    configuration, used (unless overridden) when an application, test,
    or sample is compiled for this board.

Reply:
This is verbose. Why not something like:

```
When porting Zephyr to a board, you must provide the board's default
Kconfig configuration, which is used in application builds unless explicitly
overridden.
```

Reply:
I'll take this last one, omitting "Kconfig" as it is true for any kind of configuration (pinmux, dts, ..)